crates/kitsune_p2p/kitsune_p2p/src/spawn/actor/space.rs

```diff
@@ -27,6 +27,16 @@ ghost_actor::ghost_chan! {
         /// see if an agent is locally joined
         fn is_agent_local(agent: Arc<KitsuneAgent>) -> bool;
 
+        /// Incoming Delegate Broadcast
```

Comment:
In retrospect I guess this is actually where that comment could have gone, and the other trait method comment could refer to this one, but I'm just glad that it exists somewhere now.

Reply:
coppied the comment here too :+1: